### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by 972821054** [LiuShuaiyr/UAVMOT#10]
Hello, can you provide your training model for VisDrone dataset?
Thank you very much!

**Comment by xiaocc612** [comment]
@LiuShuaiyr Hi author, do you have time to upload the pre-trained model? My retrained model performs poorly without using the triple loss reID branch. Namely, IDF1: 37.9%, MOTA: 21.0%.

**Comment by LiuShuaiyr** [comment]
It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.

**Comment by xiaocc612** [comment]
> It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.

First of all, thank you for your reply.
In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code?  I look forward to your optimized code and the sharing of pre-trained models and experimental results files.

**Comment by LiuShuaiyr** [comment]
> > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> 
> First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.

Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.

**Comment by xiaocc612** [comment]
> > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > 
> > 
> > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> 
> Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.

Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^

**Comment by LiuShuaiyr** [comment]
> > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > 
> > > 
> > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > 
> > 
> > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> 
> Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^

the test result files for reference. 
[test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)

**Comment by xiaocc612** [comment]
> > > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > > 
> > > > 
> > > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > > 
> > > 
> > > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> > 
> > 
> > Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^
> 
> the test result files for reference. [test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)

Thanks again for your quick reply and I look forward to your updates. Sorry to interrupt again, can you provide the results of the experiments on the validation set?

**Comment by lebron-2016** [comment]
> > > > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > > > 
> > > > > 
> > > > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > > > 
> > > > 
> > > > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> > > 
> > > 
> > > Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^
> > 
> > 
> > the test result files for reference. [test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)
> 
> Thanks again for your quick reply and I look forward to your updates. Sorry to interrupt again, can you provide the results of the experiments on the validation set?

大佬您好，想问下您评估成功了吗？我使用 https://github.com/VisDrone/VisDrone2018-MOT-toolkit 中的evalMOT.m文件评估自己生成的测试结果，指标显示都为0；我又评估作者提供的 test_results.zip 结果文件，指标依旧如下图所示。
![image](https://user-images.githubusercontent.com/80045218/211498374-2b1d165a-6a30-4fd9-8590-7e64d9bba8bf.png)
您有遇到过类似问题吗？特别希望得到您的解答，万分感谢！！

**Comment by xiaocc612** [comment]
> > > > > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > > > > 
> > > > > > 
> > > > > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > > > > 
> > > > > 
> > > > > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> > > > 
> > > > 
> > > > Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^
> > > 
> > > 
> > > the test result files for reference. [test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)
> > 
> > 
> > Thanks again for your quick reply and I look forward to your updates. Sorry to interrupt again, can you provide the results of the experiments on the validation set?
> 
> 大佬您好，想问下您评估成功了吗？我使用https://github.com/VisDrone/VisDrone2018-MOT-toolkit 中的evalMOT.m文件评估自己生成的测试结果，指标显示都为0；我又评估作者提供的test_results.zip结果文件，指标依旧如下图所示。 ![image](https://user-images.githubusercontent.com/80045218/211498374-2b1d165a-6a30-4fd9-8590-7e64d9bba8bf.png)

我能评估成功，你多检查下吧。

**Comment by lebron-2016** [comment]
> > > > > > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > > > > > 
> > > > > > > 
> > > > > > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > > > > > 
> > > > > > 
> > > > > > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> > > > > 
> > > > > 
> > > > > Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^
> > > > 
> > > > 
> > > > the test result files for reference. [test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)
> > > 
> > > 
> > > Thanks again for your quick reply and I look forward to your updates. Sorry to interrupt again, can you provide the results of the experiments on the validation set?
> > 
> > 
> > 大佬您好，想问下您评估成功了吗？我使用https://github.com/VisDrone/VisDrone2018-MOT-toolkit 中的evalMOT.m文件评估自己生成的测试结果，指标显示都为0；我又评估作者提供的test_results.zip结果文件，指标依旧如下图所示。 ![image](https://user-images.githubusercontent.com/80045218/211498374-2b1d165a-6a30-4fd9-8590-7e64d9bba8bf.png)
> 
> 我能评估成功，你多检查下吧。

可以麻烦您帮忙看下吗？这是我对 https://github.com/VisDrone/VisDrone2018-MOT-toolkit 中的 evalMOT.m 文件的设置， uav_test_result 是上边提供的 test_result.zip，实在不知道是哪里出了问题  ╥﹏╥...
![image](https://user-images.githubusercontent.com/80045218/211505321-1da8cdb0-78ff-4e6b-a998-06a5bb6c0079.png)

**Comment by ghost** [comment]
> > > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > > 
> > > > 
> > > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > > 
> > > 
> > > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> > 
> > 
> > Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^
> 
> the test result files for reference. [test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)

@LiuShuaiyr  Thanks for your shared inference results on VisDrone test set!

We are a research team that is studying on MOT algorithms for practical UAV scenes, and our scenes are more similar to UAVDT dataset. So we are trying to reproduce your wonderful codes on UAVDT test set, but have failed to get a promising result (compared with the results in Table 1 of your paper). Could you provide your final tracking results on UAVDT test set  for our reference? We will appreciate and cite the results and work appropriately. Thanks~

**Comment by LiuShuaiyr** [comment]
> > > > > > It is recommended to solve the training problem before testing. In addition, the following changes should be made in the track.py file, using MCJDETrcker_ AMF instead of MCJDETrcker. We will optimize this part of code later.
> > > > > 
> > > > > 
> > > > > First of all, thank you for your reply. In the training phase, just no triplet loss is used. In the inference phase, I tested MCJDETrcker_ AMF and MCJDETrcker, and the results are the same. I will double check. Maybe there are errors in the released code? I look forward to your optimized code and the sharing of pre-trained models and experimental results files.
> > > > 
> > > > 
> > > > Thank you for your question. Because the code is not well organized, the released code may have some problems. After finishing the work at hand, I will continue to improve the code and upload the result file, It will take about 3 months.
> > > 
> > > 
> > > Thank you for your quick reply. If it's not too much trouble, can you share the test result files? This should not take much time. ^_^
> > 
> > 
> > the test result files for reference. [test_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10299630/test_results.zip)
> 
> @LiuShuaiyr Thanks for your shared inference results on VisDrone test set!
> 
> We are a research team that is studying on MOT algorithms for practical UAV scenes, and our scenes are more similar to UAVDT dataset. So we are trying to reproduce your wonderful codes on UAVDT test set, but have failed to get a promising result (compared with the results in Table 1 of your paper). Could you provide your final tracking results on UAVDT test set for our reference? We will appreciate and cite the results and work appropriately. Thanks~

The UAVDT results files for reference 
[UAVDT_results.zip](https://github.com/LiuShuaiyr/UAVMOT/files/10836208/UAVDT_results.zip)

**Comment by xiaocc612** [comment]
> 
Is the training set and test set in UAVDT a random selection of 30 and 20 video sequences respectively? I have searched for a long time, but I can't find the official division. Can you tell me about it? Thank you.
